src/signals/incident/definitions/wizard-step-2-vulaan/__mocks__/afval.js

```diff
@@ -0,0 +1,51 @@
+import { Validators } from 'react-reactive-form';
```

Comment:
Where is this mock used?

Reply:
[Here](https://github.com/Amsterdam/signals-frontend/blob/01c26302656341a97bc0bf945205d21e35500f32/src/signals/incident/definitions/__tests__/wizard-step-5-samenvatting.test.js#L19)

Reply:
Not sure I understand, the `extra_bedrijven_horeca_wat` is mocked in the afval component but comes from `overlast-bedrijven-en-horeca`. Something is not matching there. Can you take a look again? 
A mock shouldn't necessary if you check on the actual values from `afval` category
```
      const expected = expect.objectContaining({
        vulaan: {
          extra_afval: {
            label: 'Waar vandaan',
            optional: true,
            render: Label,
          },
          extra_container_kind: {
            label: 'Soort container',
            optional: true,
            render: Label,
          },
          extra_container_number: {
            label: 'Container nummer',
            optional: true,
            render: Label,
          },
        },
```

Reply:
You're right, the mock isn't necessary! 👍